json/circe/src/main/scala/sttp/client/circe/SttpCirceApi.scala

```diff
@@ -8,6 +9,10 @@ import sttp.client.{IsOption, ResponseAs, ResponseError}
 import sttp.model.MediaType
 
 trait SttpCirceApi {
+  implicit val errorMessageForCirceError: ErrorMessage[io.circe.Error] = new ErrorMessage[circe.Error] {
+    override def extract(t: circe.Error): String = t.getMessage
```

Comment:
that's simply the default, so we don't really need it?

Reply:
Good point, removing it...